revolv/templates/project/edit_project.html

```diff
@@ -132,4 +145,28 @@
         });
     });
 </script>
+<!-- Initializes the category selection tool -->
+<script src="{% static 'chosen/chosen.jquery.js' %}"></script>
+<script src="{% static 'chosen/docsupport/prism.js' %}" type="text/javascript" charset="utf-8"></script>
+<script>
+    $( document ).ready(function() {
+        // populates the currently selected categories if none of them have been selected
```

Comment:
Very hacky, but it works. Not sure how to do this better, especially given how the Django template adds lists to HTML